required/latex-lab/latex-lab-math.dtx

```diff
@@ -310,11 +310,25 @@
       {
         \@@_process:nn { math } {#1} % $
 % fairly simple this one
```

Comment:
Delete this line.

Reply:
Or, this comment line should be surrounded by `%    \end{macrocode}` and `%    \begin{macrocode}`?

Reply:
> Could do that, but it is wrong.
>
> Why was nothing at all done about it?

Sorry, I deleted my comment shortly after I realized that you're right. The original comment was

> Or, this comment line should be surrounded by `%    \end{macrocode}` and `%    \begin{macrocode}`?